manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
+PackageVersion: 0.12.0
+PackageLocale: en-US
+Publisher: magic-wormhole
+PublisherUrl: https://github.com/magic-wormhole/magic-wormhole
+PublisherSupportUrl: https://github.com/magic-wormhole/magic-wormhole/issues
+# PrivacyUrl:
+Author: Brian Warner
+PackageName: magic-wormhole
+PackageUrl: https://magic-wormhole.readthedocs.io/en/latest/
+License: MIT
+LicenseUrl: https://github.com/magic-wormhole/magic-wormhole/blob/master/LICENSE
+Copyright: Copyright (c) 2015 Brian Warner
+# CopyrightUrl:
+ShortDescription: Get things from one computer to another, safely.
+# Description:
+Moniker: wormhole
+Tags:
+  - file-sharess
```

Comment:
```suggestion
  - file-shares
```

Reply:
I'm going to change the tag to `file-sharing`. That seems to be the correct tag.